xlive/Blam/Engine/effects/particle_system.h

```diff
@@ -38,7 +38,7 @@ CHECK_STRUCT_SIZE(s_particle_system_update, 0xC);
 class c_particle_system
 {
 public:
-	int32 datum_salt;
+	uint16 datum_salt;
```

Comment:
Should we define padding below this so that it's explicit?

Reply:
not necessarily